app/TestTCP/testenv.hpp

```diff
@@ -44,6 +44,10 @@ class TestEnv1 : public ::testing::Test
 
 	virtual void SetUp()
 	{
+		if(getenv("RANDOM_SEED") == NULL)
+			setenv("RANDOM_SEED", std::to_string(time(0)).c_str(), true);
+		srand(atoi(getenv("RANDOM_SEED")));
+
```

Comment:
You can have print statements here.

Reply:
As we are using c++, I prefer `cout` instead of `printf`.

Reply:
At first, my `printf` was in `Setup` function fo `testenv.hpp`, but I moved it to each test functions after first PR. I'll move it again.